ancilla.el

```diff
@@ -499,7 +517,12 @@ replacement text as argument."
    (concat "User selected the region marked by <|begin selection|>/<|end selection|> and asked:\n"
            instruction))
 
-  (ancilla--adaptor-chat-request-buffer-send 'message))
+  (ancilla--adaptor-chat-request-buffer-send (lambda (message)
+                                               ())) ;; discard the message, we can print to some buffer in future if we want
+
+  ;; show result
+  (switch-to-buffer-other-window ancilla-chat)
```

Comment:
since the answer could be long, personally feeling switch to ancilla-chat window will be much easier to read